pylint_nautobot/__init__.py

```diff
@@ -22,6 +23,7 @@
     NautobotIncorrectBaseClassChecker,
     NautobotModelLabelChecker,
     NautobotReplacedModelsImportChecker,
+    NautobotUseSearchFilterChecker,
     NautobotStringFieldBlankNull,
     NautobotSubClassNameChecker,
     NautobotUseFieldsAllChecker,
```

Comment:
alpha sort?

```suggestion
    NautobotStringFieldBlankNull,
    NautobotSubClassNameChecker,
    NautobotUseFieldsAllChecker,
    NautobotUseSearchFilterChecker,
```

Reply:
I guess Ruff isn't doing this automatically. I will alpha sort.